package.json

```diff
@@ -60,8 +60,8 @@
     "testem:ci": "testem ci"
   },
   "optionalDependencies": {
-    "canvas": "^1.6.12",
-    "jsdom": "^9.12.0",
+    "canvas": "^2.0.1",
```

Comment:
Canvas 2.0 has some [breaking changes](https://github.com/Automattic/node-canvas/blob/master/CHANGELOG.md#200)

Reply:
i know i know, did not read yet how to interact with new jsdom. Canvas is transparent for us